composer.json

```diff
@@ -41,7 +18,7 @@
     "wpackagist-plugin/pantheon-advanced-page-cache": "^0.1.4",
     "wpackagist-plugin/wp-cfm": "^1.4.5",
     "wpackagist-theme/twentyseventeen": "^1.1",
-    "pantheon-systems/wordpress": "^4.8",
+    "pantheon-systems/wordpress-composer": "dev-master",
```

Comment:
`pantheon-systems/wordpress-composer` has all of the tags from pantheon-systems/wordpress, up to 4.8, so I think that we can still use `^4.8` here.